src/metadatacache/MetadataCache.ts

```diff
@@ -18,7 +18,24 @@ export interface SearchQuery {
   text?: string
   offset?: number
   page?: number
-  query: { [property: string]: string | number | string[] | number[] }
+  query: {
+    nativeSearch?: number
```

Comment:
I think we can remove `nativeSearch?: number` completely from the search since it's no longer required or used in Aquarius. This can require some market updates but not sure about that some of the search queries in the market are no longer using search typings from ocean js and are using any as type.